ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/util/BlockBlobSidecarsTrackersPoolImpl.java

```diff
@@ -575,8 +575,8 @@ private void onFirstSeen(final SlotAndBlockRoot slotAndBlockRoot) {
                     .handleException(
                         error ->
                             LOG.warn(
-                                "Local EL blobs lookup failed: {}",
-                                ExceptionUtils.getMessage(error)))
+                                "Local EL blobs lookup failed, fetching missing blobs from remote peers: {}",
```

Comment:
Mmmh not sure if this would look good. Seems to me the error is related to the el look up but is next to the remote peers fetching msg part.

Reply:
Right, put it back as it was. At least logging the root cause message only...